### ISSUE TITLE
can you make nfs4 bindmount optional

### ISSUE THREAD

**Issue by marszip** [derdanne/puppet-nfs#90]
Hi 

We are using this module for our nfs server, but we are having problems with the bindmount of the to be exported file system. All our filesystems are on a lvm and are being managed by the puppetlabs/lvm module, but this module also mounts this filesystem. 
As a workaround a have made a copy of the export.pp and excluded the following:

```
 nfs::functions::nfsv4_bindmount { $name:
      ensure         => $ensure,
      v4_export_name => $v4_export_name,
      bind           => $bind,
    }
```
But this is far from optimal.  Is it possible to make a boolean for mounting this nfs export?

I hope this is possible.

Thanks

**Comment by derdanne** [comment]
committed in develop

**Comment by derdanne** [comment]
released in 2.1.0
